docs/installation.rst

```diff
@@ -147,12 +144,8 @@ Gunicorn, preferably in **async mode**, which allows for impressive
 concurrency even and is fairly easy to install and configure. Please
 refer to the
 documentation of your preferred technology to set up this Flask WSGI
-application in a way that works well in your environment.
-
-While the `superset runserver` command act as an quick wrapper
-around `gunicorn`, it doesn't expose all the options you may need,
-so you'll want to craft your own `gunicorn` command in your production
-environment. Here's an **async** setup known to work well: ::
+application in a way that works well in your environment. Here's an **async**
+setup known to work well in production: ::
 
  　gunicorn \
```

Comment:
We are running gunicorn with gevent which make superset more efficient handling requests `-k gevent '

Reply:
@fabianmenges the `-k gevent` argument is specified.